ui/packages/consul-ui/app/components/consul/upstream-instance/list/index.hbs

```diff
@@ -63,7 +52,18 @@ as |mode|}}
               {{item.LocalBindSocketPath}}
             </dd>
           </dl>
-{{/let}}
+        {{#if item.LocalBindSocketMode}}
+          <dl class="local-bind-mode">
+            <dt>
+              <span>
+                Local bind mode
+              </span>
+            </dt>
+            <dd>
+              <span>mode:</span>{{or item.LocalBindSocketMode '-'}}
+            </dd>
+          </dl>
```

Comment:
Looks like we have two titles here? `Local bind mode` and `mode`, could we just use one?

Reply:
yea, I'll just use `mode`.

Reply:
Not sure if I'm making sense here, with that additional change we still have two titles `Mode` and `mode`, and I think we can keep the `or` and lose a conditional.

> I think here we can just remove the if entirely?

So think of it like this maybe:

```javascript
if(primaryProperty) {
  console.log(`Primary Property: ${primaryProperty}`);
  console.log(`Secondary Property: ${secondaryProperty || '-'}`);
}
```

which could output:

```sh
Primary Property: Hello
Secondary Property: -
# or
Primary Property: Hello
Secondary Property: There
# or
..nothing
```

Happy to sync up on this if thats easier?